docs/index.html

```diff
@@ -35,7 +35,7 @@
             grid: "#bbb",
             inactive: "#ddd"
         },
-        title: "Connect Holland Tech Radar — 2021.09",
+        title: "Harborn Tech Radar — 2022.01",
```

Comment:
Is .01 the month. quarter or version?
If month, change to .02

Reply:
It's month and the one we establish in January, so 1 seems right to me.